Dockerfile

```diff
@@ -1,15 +1,14 @@
 FROM node:10-alpine
 
 # Create app directory
-WORKDIR /usr/src/makina-api/
+WORKDIR /usr/makina-api/
 
 # Install dependencies
 COPY package.json package.json
 RUN npm install --only=production
 
 # Copy app sources
 COPY dist dist
-COPY .env .env
```

Comment:
When .env is not copied, does nodejs app still work?

Reply:
.env file is not in the repository so it is not possible to include it in the artifact. but turns out it doesn't make sense to do so anyways. because, for when we run it from our local computers, dotenv will find and load our .env file, for when we run it as a container, we need to provide those env variables just like how we do with the container for mysql (like how the root password is passed inside as a parameter).

so, container as being a generic artifact, it shouldn't contain any usage specific env variables like db address or password which in the will change for everyone using the container image.

Reply:
ok, we need to update docker-compose.yml, we can do it later.